src/JITModule.cpp

```diff
@@ -223,14 +224,14 @@ JITModule::JITModule() {
 JITModule::JITModule(const Module &m, const LoweredFunc &fn,
                      const std::vector<JITModule> &dependencies) {
     jit_module = new JITModuleContents();
-    llvm::Module *llvm_module = compile_module_to_llvm_module(m, jit_module.ptr->context);
+    std::unique_ptr<llvm::Module> llvm_module(compile_module_to_llvm_module(m, jit_module.ptr->context));
```

Comment:
I'd rather wait to start pushing more use of unique_ptr through Halide. I think it is a good idea, but I also think it could happen in a fair number of places and would rather undertake that as a specific task.